docs/src/main/asciidoc/getting-started-dev-services.adoc

```diff
@@ -142,7 +142,7 @@ public class Greeting extends PanacheEntity {
 }
 ----
 
-The entity makes use of xref:{doc-guides}hibernate-orm-panache.adoc[Panache], a layer on top of Hibernate ORM.
+The entity makes use of xref:{doc-guides}/hibernate-orm-panache.adoc[Panache], a layer on top of Hibernate ORM.
```

Comment:
This is incorrect, `{doc-guides}` shouldn't have been used. Can you fix it here? And maybe check if there are other occurrences in our doc? We shouldn't use it to point to our `.adoc` files.
```suggestion
The entity makes use of xref:hibernate-orm-panache.adoc[Panache], a layer on top of Hibernate ORM.
```

Reply:
Happy to fix those yes. I replaced `docfile` with `doc-guides` because these seemed to work, so there are a few more